helm/logging/templates/fluent-bit-configmap.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
```

Comment:
What's the requirement for having this be checked in vs. just using what's provided by the stable fluentbit chart?

Reply:
agreed. i will test/verify.